File: Java/poo/gymRegistration/library/basicPlan.java
package poo.gymRegistration.library;

    public class basicPlan extends user{
        private final double cost = 60000.0; 

    public basicPlan(String names, String lastnames, int identification, double weight, double height) {
        super(names, lastnames, identification, weight, height);
    }

    public void getShowServices(){
        System.out.println("Plan Basico - Servicios:");
        System.out.println( "Acceso General al Gym ");
    }

    public double getCost(){
        return this.cost;
    }
}

File: Php/poo/gymRegistration/library/basicPlan.php
<?php

    class basicPlan extends user{
        private float $cost = 60000.0; 
        private string $showServices;

    public function __construct(string $names, string $lastnames, int $identification, float $weight, float $height) {
        parent::__construct($names, $lastnames, $identification, $weight, $height);
    }

    public function showServices(): void {
        echo "Plan Basico - Servicios:<br>";
        echo " Acceso General al Gym <br>";
    }

    public function getCost(): float {
        return $this->cost;
    }
}